packages/storybook/README.md

```diff
@@ -0,0 +1,72 @@
+# @builder.io/storybook
+
+Addon for integrating [Builder.io](https://www.builder.io) to allow drag and drop page building inside storybook. 
+
+<img src="https://user-images.githubusercontent.com/5093430/76154244-ebbe6480-608d-11ea-9dc9-08a59eda220e.gif" alt="example" />
```

Comment:
we should also update the react-esign-system readme to mention this shows storybook too (linking to this readme and to the hosted storybook for this we should add)